client/my-sites/people/main.jsx

```diff
@@ -55,6 +55,21 @@ class People extends React.Component {
 		}
 	}
 
+	renderSubheaderText() {
+		const { translate, filter } = this.props;
+
+		switch ( filter ) {
+			case 'followers':
+				return translate(
+					'People who have subscribed to your site using their WordPress.com account.'
```

Comment:
Can these be full sentences like the default, eg. "Manage people who have..."

Reply:
Thoughts on this /cc @donalirl @lauac ?

Reply:
This didn't come up during the review with Editorial. I see the reasoning: every subtitle starts with a verb, except for Followers, Email Followers, and My Home. I'm not sure I want to lead with "Manage" for the Followers & Email Followers screens. The primary use of these screens is to _view_ your followers, and manage/removing is a secondary use. This is what I'm thinking: 

View and manage the people who have subscribed to your site using their WordPress.com account.
View and manage the people who have subscribed to your site using their email address.

Reply:
Oh, I'm happy to defer to Editorial, it just thought it might be an oversight.